docs/Customization/Dsptools-Blocks.rst

```diff
@@ -0,0 +1,105 @@
+.. _dsptools-blocks:
```

Comment:
@grebe , I think there should be some introduction to DSP tools. I know there is some in the dsp-tools README, but I don't think it's sufficient.

This page talks about Ring operations which could use some additional introduction